src/appMain/sdl_preloaded_pt.json

```
@@ -3859,25 +3859,63 @@
                             "name": "driverDoorAjar",
                             "key": "OEM_REF_DR_DOOR_AJ",
                             "type": "Boolean",
-                            "mandatory": false
+                            "mandatory": false,
+                            "deprecated": true,
+                            "since": "7.1"
                         },
                         {
                             "name": "passengerDoorAjar",
                             "key": "OEM_REF_PAS_DOOR_AJ",
                             "type": "Boolean",
-                            "mandatory": false
+                            "mandatory": false,
+                            "deprecated": true,
+                            "since": "7.1"
                         },
                         {
                             "name": "rearLeftDoorAjar",
                             "key": "OEM_REF_REAR_LEFT_DOOR_AJ",
                             "type": "Boolean",
-                            "mandatory": false
+                            "mandatory": false,
+                            "deprecated": true,
+                            "since": "7.1"
                         },
                         {
                             "name": "rearRightDoorAjar",
                             "key": "OEM_REF_REAR_RIGHT_DOOR_AJ",
                             "type": "Boolean",
-                            "mandatory": false
+                            "mandatory": false,
+                            "deprecated": true,
+                            "since": "7.1"
+                        },
+                        {
+                            "name": "doorStatuses",
+                            "key": "OEM_DOOR_STATUSES",
```

@LitvinenkoIra I propose to keep consistency in keys naming and use `OEM_REF_` prefix as its done for the rest:
```suggestion
                            "key": "OEM_REF_DOOR_STATUSES",
```

---

@AKalinich-Luxoft please check ff3c1607c97bd6a2388ef9c836de1452b59d629e